src/engine/effects/engineeffectsdelay.cpp

```diff
@@ -0,0 +1,96 @@
+#include "engine/effects/engineeffectsdelay.h"
+
+#include "util/rampingvalue.h"
+#include "util/sample.h"
+
+namespace {
+// See enginedelay.cpp
+constexpr int kMaxDelay = static_cast<int>(0.508 *
+        mixxx::audio::SampleRate::kValueMax * mixxx::kEngineChannelCount);
```

Comment:
Lets just remove this constant for now.
```suggestion
constexpr int kMaxDelay = mixxx::audio::SampleRate::kValueMax * mixxx::kEngineChannelCount;
```